Storage/Bookmarks/Bookmarks.swift

```diff
@@ -14,6 +14,10 @@ public protocol SearchableBookmarks: class {
     func bookmarksByURL(_ url: URL) -> Deferred<Maybe<Cursor<BookmarkItem>>>
 }
 
+public protocol CountableBookmarks: class {
+    func numberOfBookmarks() -> Deferred<Maybe<Int>>
```

Comment:
Sounds good - I'll push up a pass on the query. I'd imagine it would be similar to how we're pulling in bookmarks in our panel view without the trimming of the result set by folder?